### PR TITLE
docs: mention builtin commenting in :h commentstring

### PR DESCRIPTION
**Pull request by tom-anders**

### CODE REVIEW

**Comment by tom-anders** [comment]
(I took the liberty to assume that built-in commenting is probably more important than folding to the average user)

**Comment by tom-anders** [comment]
@echasnovski FYI

File: src/nvim/options.lua [hunk 1]
```diff
@@ -1323,8 +1323,9 @@ return {
       defaults = { if_true = '' },
       desc = [=[
         A template for a comment.  The "%s" in the value is replaced with the
-        comment text.  For example, C uses "/*%s*/". Currently only used to
-        add markers for folding, see |fold-marker|.
+        comment text. For example, C uses "/*%s*/". Most importantly, this
+        is used for commenting and uncommenting, see |commenting|.
+        Also used to add markers for folding, see |fold-marker|.
```
Comment (echasnovski) [inline src/nvim/options.lua:1328]:
```suggestion
        comment text. For example, C uses "/*%s*/". Used for |commenting| and to
        add markers for folding, see |fold-marker|.
```

I think being as concise as possible is usually the goal in built-in docs.

**Review by echasnovski — CHANGES_REQUESTED** [review]
Nice catch! Should have been updated in original PR.

**Comment by zeertzjq** [comment]
Need to run `make doc` again

**Comment by dundargoc** [comment]
Thank you for your contribution! We collect all typo fixes and similar small fixes into a single pull request and merge it once it gets big enough: https://github.com/neovim/neovim/pull/28208

**Pull request closed without merging**